### DISCUSSION TITLE
Catching up on pages that could not be imported

### DISCUSSION THREAD

**Discussion by bgorath** [WikiTransformationProject/wikitraccs-releases#19] [General]
Hello, 

during my tests I decided to restart Confluence (not intentionally). Wikitraccs moved forward and just skipped a few pages. 

After finishing all exptected pages of the targeted confluence space, there are still four pages missing. How can I restart import for just those missing pages without having to start the migration all over again?

Thanks!

**Comment by heinrich-ulbricht** [comment]
Hi! Just start again normally. By default WikiTraccs skips all pages that are already there in SharePoint. So it should just migrate the few missing.
